test/restClient.spec.js

```diff
@@ -0,0 +1,228 @@
+const expect = require('chai').expect;
```

Comment:
Thanks for the feedback, will fix it now...It doesn't show any errors for me though, does it have anything to do with node version?